packages/history-sdk/README.md

```diff
@@ -1,4 +1,4 @@
-# Typescript SDK for commercetools machine learning API
+# Typescript SDK for commercetools audit log APIs.
```

Comment:
the term "audit log" should be capitalized as "Audit log"

Reply:
```suggestion
# Typescript SDK for commercetools Audit log APIs.
```